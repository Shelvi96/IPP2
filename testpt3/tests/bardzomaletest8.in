NEW ID

78080288>2:8
261948;66>05:9:
0008>907
070;>61572931
118:>9
9>09:5::08
;0>:677198
:>93
7713>28:
183586880>6757817
@410;15797370847501115;7
@8789265
@;5:0:5::05:81516:673
@6755
@68507495
@83763084615034;:159;345;
@5:533236912296823897;71840
@09:170::8156:680720168:5;5:892
@10:27:0965618490;410212;17
@984498237578;52416619:6
@48339:5::26;:9
@66265;9203855:0620:
@103;35527695030:29064039
@340369464799149;5
@406263600:718;98639322:333;
@747;815
@936112295
@9;76698954444
