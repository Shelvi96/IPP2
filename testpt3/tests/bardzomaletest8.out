3486784401
0
9
0
0
285331494525
108484459536384
102010210000000000
35095734178747
26366172976
9
544644
285331172423
6561
386026904695564
0
0
1
